web/client/components/map/openlayers/plugins/VectorLayer.js

```diff
@@ -106,23 +106,7 @@ var styleFunction = function(feature) {
 
 Layers.registerType('vector', {
     create: (options) => {
-        let features;
```

Comment:
Uhm, there needs to be a way to suppress this though, since when adding a vector layer to the layers state, Feature.jsx objects are created automatically. This leads to those Feature.jsx objects being correctly managed whereas those added directly are unmanaged, meaning they are not removed anymore when changing the layer properties (i.e. changing the `features` field) without removing and re-adding the layer. I don't know the JSON use-case, do you see a solution?